package.json

```diff
@@ -59,20 +59,22 @@
     "react-dom": ">=15.4.1 <16.9.0 || 16.7.0-alpha.0"
   },
   "devDependencies": {
+    "acorn": "^6.0.5",
+    "@babel/core": "^7.4.3",
     "@babel/generator": "^7.2.2",
     "@babel/types": "^7.2.2",
     "@commitlint/cli": "7.2.0",
     "@commitlint/config-conventional": "7.1.2",
     "@dcos/tslint-config": "0.1.1",
-    "@storybook/addon-a11y": "3.4.11",
-    "@storybook/addon-actions": "3.4.11",
-    "@storybook/addon-info": "3.4.11",
-    "@storybook/addon-knobs": "3.4.11",
-    "@storybook/addon-links": "3.4.11",
-    "@storybook/addon-options": "3.4.11",
-    "@storybook/addons": "3.4.11",
-    "@storybook/react": "3.4.11",
-    "@storybook/storybook-deployer": "2.3.0",
+    "@storybook/addon-a11y": "5.0.5",
+    "@storybook/addon-actions": "5.0.5",
+    "@storybook/addon-info": "5.0.5",
+    "@storybook/addon-knobs": "5.0.5",
+    "@storybook/addon-links": "5.0.5",
+    "@storybook/addon-options": "5.0.5",
+    "@storybook/addons": "5.0.5",
+    "@storybook/react": "5.0.5",
```

Comment:
I originally tried to upgrade to Storybook 5.0.6 (latest stable release), but had troubles due to conflicting versions of `emotion`. We use `emotion` 9, Storybook is on `emotion` 10.

If we can get @Poltergeist's emotion 10 PR (https://github.com/dcos-labs/ui-kit/pull/314) merged first, we can to 5.0.6 instead